api/envoy/service/status/v3/csds.proto

```diff
@@ -114,6 +114,8 @@ message PerXdsConfig {
     admin.v3.ScopedRoutesConfigDump scoped_route_config = 5;
 
     admin.v3.EndpointsConfigDump endpoint_config = 6;
+
+    admin.v3.GenericXdsConfigDump generic_xds_config_dump = 1000;
```

Comment:
just use the next free ID, which is 8, instead of 1000.

Reply:
This is removed.